google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java

```diff
@@ -775,6 +790,189 @@ public void awaitReplication(String tableId) {
         stub.awaitReplicationCallable().futureCall(tableName));
   }
 
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public Backup createBackup(CreateBackupRequest request) {
+    return ApiExceptions.callAndTranslateApiException(createBackupAsync(request));
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public ApiFuture<Backup> createBackupAsync(CreateBackupRequest request) {
+    return ApiFutures.transform(
+        stub.createBackupOperationCallable().futureCall(request.toProto(projectId, instanceId)),
+        new ApiFunction<com.google.bigtable.admin.v2.Backup, Backup>() {
+          @Override
+          public Backup apply(com.google.bigtable.admin.v2.Backup backupProto) {
+            return Backup.fromProto(backupProto);
+          }
+        },
+        MoreExecutors.directExecutor());
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public Backup getBackup(String clusterId, String backupId) {
+    return ApiExceptions.callAndTranslateApiException(getBackupAsync(clusterId, backupId));
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public ApiFuture<Backup> getBackupAsync(String clusterId, String backupId) {
+    GetBackupRequest request =
+        GetBackupRequest.newBuilder()
+            .setName(NameUtil.formatBackupName(projectId, instanceId, clusterId, backupId))
+            .build();
+    return ApiFutures.transform(
+        this.stub.getBackupCallable().futureCall(request),
+        new ApiFunction<com.google.bigtable.admin.v2.Backup, Backup>() {
+          @Override
+          public Backup apply(com.google.bigtable.admin.v2.Backup backup) {
+            return Backup.fromProto(backup);
+          }
+        },
+        MoreExecutors.directExecutor());
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public List<String> listBackups(String clusterId) {
+    return ApiExceptions.callAndTranslateApiException(listBackupsAsync(clusterId));
+  }
+
+  @BetaApi("Managed Backup feature is in alpha and may change in the future.")
+  public ApiFuture<List<String>> listBackupsAsync(String clusterId) {
+    ListBackupsRequest request =
+        ListBackupsRequest.newBuilder()
+            .setParent(NameUtil.formatClusterName(projectId, instanceId, clusterId))
+            .build();
+
+    // TODO(igorbernstein2): try to upstream pagination spooling or figure out a way to expose the
```

Comment:
@igorbernstein2 still relevant?

Reply:
I'd like to extract this to a helper at the very least